_data-prepper/pipelines/configuration/processors/parse-json-processor.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Parse_JSON
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Parse JSON
+
+The `Parse_json` processor parses JSON data for an event, including any nested fields. The processor extracts the JSON pointer data and adds the input event to the extracted fields.
+
+
+## Configuration
+
+You can configure the `Parse_JSON` processor with the following options.
+
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `Event` that will be parsed. Default value is `message`. |
```

Comment:
Do we need to list other values for this option, aside from he default? Or are they obvious to the user?